extensions/graphql/src/__tests__/fixtures/graphql-resolvers/recipe/recipe.model.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright IBM Corp. 2021. All Rights Reserved.
+// Node module: @loopback/graphql
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {field, ID, objectType} from '../../../../';
+@objectType({description: 'Object representing cooking recipe'})
+export class Recipe {
+  @field(type => ID)
+  id: string;
+
```

Comment:
Since we are using the powerful `graphql-tools` can you add a custom directive to leverage the use case for `printSchemaWithDirectives` method?

Reply:
I'll admit that I'm not familiar with custom directives- I was simply following the TypeGraphQL guide for printing the spec.  I'll do some research and ensure that a custom directive is included in the spec.